docs/resources/azure_virtual_network_peerings.md

```diff
@@ -0,0 +1,80 @@
+---
+title: About the azure_virtual_network_peerings Resource
+platform: azure
+---
+
+# azure_virtual_network_peerings
+
+Use the `azure_virtual_network_peerings` InSpec audit resource to test properties related to virtual network peerings of a virtual network.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
```

Comment:
Interacts is a vague word. Consider replacing with `uses` or another precise term.